### PR TITLE
Add testing for Phantompeakqualtools 

### PR DESCRIPTION
**Pull request by sruthipsuresh**
I've updated the testing (workflow/main.nf/test.yml) for the Phantompeakqualtools module to follow the pytest-workflow format - this addresses #98.
Please let me know if there is anything which needs to be fixed!

### CODE REVIEW

**Comment by KevinMenden** [comment]
Couldn't spot any errors, and since the test passes as well, this looks pretty good!

Only thing I was wondering is why you removed the md5 sums for pdf and rdata? Are those files not reproducible?

**Review by KevinMenden — COMMENTED** [review]
Couldn't spot any errors, and since the test passes as well, this looks pretty good!

Only thing I was wondering is why you removed the md5 sums for pdf and rdata? Are those files not reproducible?

**Comment by Emiller88** [comment]
@KevinMenden The md5 sums change for pdfs and Rdata, my guess is because they may have timestamp in them. We may try to use the contains option in the future to get the important parts out of it.

**Comment by KevinMenden** [comment]
I see, thanks for the explanation @Emiller88  :)

**Review by KevinMenden — APPROVED** [review]
Looks good to me and all checks pass - so I say go ahead an merge :) 

👍

**Comment by Emiller88** [comment]
@KevinMenden Thanks! Our thought process on these to just get the tests started, and if anyone finds that the output from the tests isn't what's expected then they can open an issue.

File: .github/workflows/phantompeakqualtools.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - software/phantompeakqualtools/**
+      - .github/workflows/phantompeakqualtools.yml
+      - tests/software/phantompeakqualtools/**
+
+jobs:
+  ci_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        nxf_version: [20.07.1, ""]
```
Comment (drpatelh) [inline .github/workflows/phantompeakqualtools.yml:19]:
```suggestion
        nxf_version: [20.11.0-edge]
```

**Review by drpatelh — APPROVED** [review]
Thanks guys!!

File: .github/workflows/phantompeakqualtools.yml (hunk 1 above)
Comment (Emiller88) [reply .github/workflows/phantompeakqualtools.yml]: Good catch thanks!

**Review by Emiller88 — APPROVED** [review]

**Pull request merged**